tools/tsp-client/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release
 
+## Unreleased - 0.14.0
+
+- Fix `init` command when using a local spec:
```

Comment:
alignment issue?

Reply:
Did it on purpose, let me check if it looks good